docs/guides/setting-up-a-networked-meta-avatar.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: docs
+title: Setting up a Networked Meta Avatar
+description: This guide walks you through setting up a networked meta avatar.
+
+---
+
+# Setting up a Networked Meta Avatar
+
+This guide will demonstrate how to quickly load a Meta Avatar and network it to other machines using Normal.
+
+When you're finished, you'll end up with an application that will let you see and talk to other player's avatars:
+
+![](./setting-up-a-networked-meta-avatar/player-wave.png "Meta Avatar waving at the viewer") 
+
+## Setup your App for Meta Avatars
+
+Before your app can load avatar data from Meta's servers, it will need permission to do so.  Ensure that your app is [configured as required for Meta Avatar usage](https://developer.oculus.com/documentation/unity/meta-avatars-app-config/), and that you have set your Oculus Platform Settings to point at the right app IDs
```

Comment:
The linked Meta documentation doesn't show how to get the app ID and configure Oculus Platform Settings for the project. I don't think it's enough to say "that you have set your Oculus Platform Settings to point at the right app IDs". Even as a seasoned developer, I don't know what that means unless I've done it before.

Reply:
Added a bunch of documentation around navigating the Meta Developer website and setting up your app ids and permissions (and some notes for Normcore as well).